Makefile

```diff
@@ -319,6 +319,16 @@ azure-secrets:
 	oc create secret generic codecov-token --from-literal=upload=${CODECOV_UPLOAD_TOKEN} -o yaml --dry-run | oc apply -n azure -f -
 .PHONY: azure-secrets
 
+vsphere-secrets:
+	oc create secret generic cluster-secrets-vsphere \
+	--from-file=cluster/test-deploy/vsphere/.awscred \
+	--from-file=cluster/test-deploy/vsphere/ssh-privatekey \
+	--from-file=cluster/test-deploy/vsphere/ssh-publickey \
```

Comment:
Why are you using the SSH keypair for AWS?

Reply:
A single keypair would make debugging easier. I don't mind using a different keypair though